code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -19,7 +19,7 @@
 	name = "carp fillet"
 	desc = "A fillet of spess carp meat."
```

Comment:
Rename so its cazadore related, and ill approve

Reply:
Done and done